### DISCUSSION TITLE
how to produce the sense of the motion along the fracturing features

### DISCUSSION THREAD

**Discussion by Viecgg** [gmtsar/gmtsar#283] [Q&A]
Dear Eric Xu,
     it is amazing to see so clearly the sense of the motion along the fracturing features as the Figure 3c-d in your Ridgecrest article.
![image](https://user-images.githubusercontent.com/19190139/155276727-9a3e5312-1215-44a2-b353-f7e67a36997f.png)
    I want to do same thing in my work,so  I use two Sentinel-1 tracks to decompose  the LoS deformation into east-west and up-down(south-north) deformation,then I  am not sure how to establish the sense of  the motion along the fracturing features using the decomposed deformation.
    I am afraid I didn't understand your article, so could you tell me what data should I prepare and what method I apply to produce the sense of the motion along the fracturing features as the Figure 3c-d in your article.

Yours,
Xia Tao

**Comment by Xiaohua-Eric-Xu** [comment]
I think the final step is to do a high-pass filtering. We used 800-m low pass filter and then differentiated it from the original decomposed deformation map.
